src/Console/FieldCommand.php

```diff
@@ -36,7 +36,7 @@ protected function getDefaultNamespace($rootNamespace): string
     /** @return array{string, string} */
     protected function nameParts(): array
     {
-        $name = $this->argument('name');
+        $name = (string) $this->argument('name');
```

Comment:
Looks like Larastan/PHPStan is going back and forth on what type this call is supposed to return. You can check the change history of this file, i think we used to check for `string`.

Reply:
Locally this issue seems to be gone, might be caused by the referenced PRs messing something up.